client-app/ui-kit/components/pages/matcher.vue

```diff
@@ -81,30 +93,20 @@ const seoInfo = asyncComputed<TResult | undefined>(
       return undefined;
     }
 
-    try {
-      // Load page by slug
-      const page = await innerFetch<TPageInfo>(`/storefrontapi/content/pages`, "POST", { permalink: `/${slug}` });
-
-      if (page) {
-        return {
-          page,
-        };
-      }
-    } catch (err) {
-      // If no page found load entity by slug
-      const resultItems = await innerFetch<TEntityInfo[]>(`/storefrontapi/seoInfos/${slug}`);
-
-      if (resultItems && resultItems.length) {
-        const entity = resultItems.find(
-          (item) =>
-            item.isActive && item.language.cultureName === currentLanguage.value!.cultureName && item.slug === slug
-        );
-        if (entity) {
-          return {
-            entity,
-          };
-        }
-      }
+    // If no page found load entity by slug
+    const result = await innerFetch<TSlugInfoResult>(
+      `/storefrontapi/slug/${slug}?culture=${currentLanguage.value!.cultureName}`
+    );
+
+    if (result.contentItem && result.contentItem.type === "page") {
```

Comment:
May be rewritten in more short variant:
`if (result.contentItem?.type === "page") {`